lib/sensors/src/MoistureSensor.cpp

```diff
@@ -53,7 +52,7 @@ uint16_t MoistureSensor::Read_Moisture() {
 
   char buf[2];
 
-  uint16_t ret = 65535;
+  sensorReading = 65535;
```

Comment:
why do we do this? if the read fails dont touch sensorReading

Reply:
I'm not sure if theres a way to check if the I2C read succeeded other than reading what was returned.

Reply:
make a temportary variable or something.

Reply:
ah I think I understand what you mean now